src/Directory.Build.props

```diff
@@ -2,21 +2,20 @@
     <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
     <PropertyGroup>
-        <Description>FsToolkit.ErrorHandling is a utility library to work with the Result type in F#, and allows you to do clear, simple and powerful error handling.</Description>
-        <PackageLicenseFile>License</PackageLicenseFile>
+        <Description>FsToolkit.ErrorHandling is an extensive utility library based around the F# Result type, enabling consistent and powerful error handling.</Description>
+        <Authors>demystifyfp, TheAngryByrd</Authors>
+        <Copyright>Copyright © 2018-23</Copyright>
+        <PackageProjectUrl>https://demystifyfp.gitbook.io/fstoolkit-errorhandling</PackageProjectUrl>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageReadmeFile>README.md</PackageReadmeFile> <!--https://docs.microsoft.com/en-gb/nuget/reference/msbuild-targets#packagereadmefile -->
     </PropertyGroup>
     <ItemGroup>
         <None Include="$(MSBuildThisFileDirectory)../README.md" Pack="true" PackagePath="/"/>
-        <None Include="$(MSBuildThisFileDirectory)..\License" Pack="true" PackagePath=""/>
     </ItemGroup>
 
     <PropertyGroup>
-
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
-        <IsPackable>true</IsPackable>
-        <IsTestProject>false</IsTestProject>
-        <PackageTags>f#;fsharp;railway oriented programming;ROP;result type;error handling</PackageTags>
-        <PackageReadmeFile>README.md</PackageReadmeFile> <!--https://docs.microsoft.com/en-gb/nuget/reference/msbuild-targets#packagereadmefile -->
+        <PackageTags>f#, fsharp, railway oriented programming, ROP, result type, error handling</PackageTags>
         <!-- Sourcelink -->
         <PublishRepositoryUrl>true</PublishRepositoryUrl>
```

Comment:
These settings can be consolidated via `DotNet.ReproducibleBuilds` which @baronfel introduced me to, e.g.  https://github.com/jet/equinox/blob/master/Directory.Build.props#L19C39-L19C57 

But that's a step too far for this PR/batch of changes in this space